tests/migrations/test_exceptions.py

```diff
@@ -0,0 +1,8 @@
+from django.db.migrations.exceptions import NodeNotFoundError
+from django.test import SimpleTestCase
+
+
+class ExceptionTests(SimpleTestCase):
+    def test_node_not_found_error_repr(self):
+        node = ('some_app_label', 'some_migration_label')
+        repr(NodeNotFoundError('some message', node))
```

Comment:
Might as well verify the output with `assertEqual` too.
It would be appropriate to have a Trac ticket too.